app/assets/stylesheets/components/search.css

```diff
@@ -0,0 +1,7 @@
+.c-search .c-loader {
+  display: none;
+}
+
+.c-search.loading .c-loader {
```

Comment:
`loading` class should change to `is-loading` Because we should always add `is` at the start of the class of status. Like `is-active`, `is-dragging`